demo/androidDev/build.gradle

```diff
@@ -0,0 +1,18 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+        // maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.1.3'
+        // classpath "gradle.plugin.org.jsonschema2dataclass:jsonschema2dataclass:2.1.0"
```

Comment:
and remove this maven plugin. 
How to use checked in repository plugin ?

Reply:
Only this `classpath` line breaks possibility of usage `includeBuild` in `settings.gradle`. The rest could stay the same as it is in pure demo

Reply:
When I reintroduce this line 
`classpath "gradle.plugin.org.jsonschema2dataclass:jsonschema2dataclass:2.1.0"`
I run into 

<img width="947" alt="image" src="https://user-images.githubusercontent.com/3314607/112185686-c2bd3300-8c00-11eb-8adb-aadc4feb47b9.png">

I guess I need to point anyhow to `'../..'` But how ?